content/en/cosign/overview.md

```diff
@@ -15,98 +15,96 @@ features:
 
 ![Cosign Overview](/sigstore_cosign-horizontal-color.svg)
 
-Cosign supports container signing, verification, and storage in an OCI registry.
-Cosign aims to make signatures invisible infrastructure.
+## Getting Started (Quick Start)
 
-![Cosign demo gif](/cosign.gif)
+Cosign is part of the Sigstore project. Join us on our [Slack channel](https://sigstore.slack.com/). (Need an [invite](https://links.sigstore.dev/slack-invite)?)
 
-Cosign supports:
+### Installation
 
-<list :items="features" type="info"></list>
+To use Sigstore, you need to install Cosign. Instructions to install Cosign can be found on the [Cosign Installation page](https://docs.sigstore.dev/cosign/installation/). 
```

Comment:
To me, it sounds like this is privileging Cosign above the other pieces of Sigstore, maybe it can be more oriented to what Cosign achieves?

```suggestion
To sign software artifacts and verify signatures using Sigstore, you need to install Cosign. Instructions to install Cosign can be found on the [Cosign Installation page](/cosign/installation/). 
```